install/aptitude-wrapper.sh

```diff
@@ -39,7 +39,7 @@ do
 done
 
 # Check if the current available disk size is enough.
-while [ "$(available_disk_size)" -lt "$size" ]; do
+while [ "${reset}$(available_disk_size)" -lt "$size" ]; do
```

Comment:
Added ${reset} to make sure that both the strings are white, plain. Previous comparison was made between bold blue and normal white strings, hence throwing an Error.